src/app/stJude/st-jude-toaster.scss

```diff
@@ -0,0 +1,73 @@
+.st-jude-toaster {
+  &.maximized {
+    padding: 1.5rem;
+    width: 400px;
+    border-top-left-radius: 1rem;
+    border-top-right-radius: 1rem;
+    font-size: 1rem;
+
+    .st-jude-info.visible {
+      opacity: 1;
+    }
+  }
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 10;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  background-color: #111;
+  padding: 2px;
+  p {
+    font-size: 1 rem;
+  }
+
+  transition: opacity 0.5s ease-in-out;
+}
+
+.store-buckets {
+  margin-bottom: 60px;
+}
+
+.android .store-buckets,
+.ios .store-buckets {
+  margin-bottom: 0;
+}
```

Comment:
Might be copy/paste, I don't think you need any store-buckets stuff.

Reply:
I need to add the margin on the buckets so the ad doesn't block access to items, and then zero out that margin if the user is on mobile.  An alternative is to wedge a component in the footer that grows and shrinks in the inventory view.